### PR TITLE
Use locking, some cleanup

### PR DESCRIPTION
**Pull request by bserdar**

### CODE REVIEW

File: migrator/src/main/java/com/redhat/lightblue/migrator/AbstractController.java [hunk 1]
```diff
@@ -17,6 +18,8 @@
 public abstract class AbstractController extends Thread {
 
     private static final Logger LOGGER=LoggerFactory.getLogger(AbstractController.class);
+
+    private final String callerId=UUID.randomUUID().toString();
```
Comment (dcrissman) [inline migrator/src/main/java/com/redhat/lightblue/migrator/AbstractController.java]: Locking will use a random UUID by default. No reason to duplicate that effort.

Comment (bserdar) [inline migrator/src/main/java/com/redhat/lightblue/migrator/AbstractController.java]: Forgot that it does. Will remove.

File: migrator/src/test/resources/simplelogger.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
 org.slf4j.simpleLogger.defaultLogLevel=ERROR
-org.slf4j.simpleLogger.log.com.redhat.lightblue.migrator=ERROR
+org.slf4j.simpleLogger.log.com.redhat.lightblue.migrator=DEBUG
```
Comment (dcrissman) [inline migrator/src/test/resources/simplelogger.properties]: I assume this was for testing purposes, but we probably don't actually want it this way.

Comment (bserdar) [inline migrator/src/test/resources/simplelogger.properties]: These are fixed now.

**Pull request merged**